sections/Pipeline.tex

```diff
@@ -0,0 +1,34 @@
+\section{Method}\label{sec:method}
+This section describes our method on an abstract level.
+Each phase of the method is described later on in more detail. 
+
+We start with a dataset, in our case consisting of news articles from the media group Nordjyske. 
+Their primary focus is to maintain a variety of local newspapers within the North Jutland region of Denmark. 
+The data ranges from 2017, where a total of $\sim$~63.000 articles have been extracted from their database.
+
+\subsection*{Step 1: Preprocessing Phase}
+This phase applies different \gls{NLP} methods, such as stemming and removing stop words, to simplify the data set and remove redundant information.
+Details of this phase are given in \autoref{sec:prepro}.
+After finishing this phase, we are left with $\sim$~32.000 articles, which will be used in the next step.
+
+\subsection*{Step 2: LDA}
+We train a \acrfull{lda} model on the dataset to generate topics based on the content of articles within the dataset. 
+We describe the investigation and selection of hyper parameters in \autoref{subsec:hyperparameters}. 
+After the model has been trained, we have a document-topic distribution matrix $\theta$ and a topic-word distribution matrix $\beta$.
```

Comment:
Maybe this step should be generalized into 'IR methods' or 'Train IR methods', as the other methods are kinda just scrapped in this pipeline.
Also should reference future sections.

Reply:
```suggestion
\subsection*{Step 2: LDA}
\todo[inline]{change to be about all the IR methods}
We train a \acrfull{lda} model on the dataset to generate topics based on the content of articles within the dataset. 
We describe the investigation and selection of hyper parameters in \autoref{subsec:hyperparameters}. 
After the model has been trained, we have a document-topic distribution matrix $\theta$ and a topic-word distribution matrix $\beta$.
```